cmd/import_candidates_cmd.go

```diff
@@ -29,103 +31,162 @@ func WireUpImportCandidate(c *cli.Context) (ds *clientbrownfield.ImportCandidate
 	return ds, f
 }
 
-// ImportCandidateServerImport subcommand function
-func ImportCandidateServerImport(c *cli.Context) error {
+func checkCloudAccountImportingState(c *cli.Context, cloudAccount *types.CloudAccount, state string) *types.CloudAccount {
+	debugCmdFuncInfo(c)
+	cloudAccountSvc, formatter := WireUpBrownfieldCloudAccount(c)
+
+	log.Info("Brownfield cloud account ID... ", cloudAccount.ID)
+	log.Info("Checking importing process... ")
+	for {
+		ca, err := cloudAccountSvc.GetBrownfieldCloudAccount(c.String("id"))
+		if err != nil {
+			formatter.PrintFatal("Couldn't get cloud account data", err)
+		}
+		if (cloudAccount.State != ca.State) || (ca.State != state) {
+			if ca.State == "idle" && ca.ErrorEventID != "" {
+				log.Error("Error while importing: ", ca.ErrorEventID)
+			} else {
+				log.Info("Done!")
+			}
+			return ca
+		}
+		time.Sleep(5 * time.Second)
+	}
+}
+
+// ImportCandidateServers subcommand function
+func ImportCandidateServers(c *cli.Context) error {
```

Comment:
We should not longer talk about candidates: the action now is ImportServers. This applies to other functions below and to the name of the file.